src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java

```diff
@@ -8,65 +8,68 @@
  *
  * @author Joel Rodrigo Rojas Roman
  */
-
 package org.fundacion.jala.converter.view.compiler;
 
 import javax.swing.JTabbedPane;
 import javax.swing.JPanel;
 import javax.swing.JLabel;
+import javax.swing.JOptionPane;
 import java.awt.Dimension;
 import java.awt.FlowLayout;
 import java.awt.Font;
-import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.event.ActionEvent;
 
+/**
+ * This class customizes a tabbed pane with custom tabs.
+ */
 public class ProjectTab extends JTabbedPane implements ActionListener {
-    public static int contador;
-    PlusButton button;
+    public static int COUNTER;
```

Comment:
Is COUNTER a constant? on line 57 it increases its value, maybe it should be a variable and use lower camel case.